pkg/controllers/disruption/custom_expiration.go

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
I'm anticipating that we probably don't want to do something as verbose as `karpenter.sh/voluntarily-disrupted=drifted`. Realistically, it should probably something simple like `karpenter.sh/disrupt=true` since we've already got an annotation that's `karpenter.sh/do-not-disrupt=false`